.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,19 +6,30 @@ labels: bug
 assignees: ''
 ---
 
-| Wazuh | Elastic | Rev |
-| ----- | ------- | --- |
-| 3.x   | 6.x     | 444 |
+| Wazuh | Elastic | Rev | Security |
+| ----- | ------- | --- | -------- |
+| 3.x   | 6.x     | 444 | Basic, ODFE, Xpack |
```

Comment:
we should update de default versions to 4.x 7.x 4xxx

Reply:
fixed!